standards/storing-source-code.md

```diff
@@ -40,4 +40,4 @@ the repository.
 
 ### Private repositories
 
-By default, any [ministryofjustice organisation](https://github.com/ministryofjustice) member can view private repositories. Unless explicitly and strictly defined on a per-repository basis to override default permissions: private repositories should be considered "organisation public" as opposed to "private" to your team/project.
+By default, any [ministryofjustice organisation](https://github.com/ministryofjustice) member has no additional privileges. This means they can clone and pull public repos only. Unless explicitly and strictly defined on a per-repository basis to override default permissions: internal/private repositories should be considered "private" to your team/project.
```

Comment:
Any member of the organisation can view an "internal" repo - for example when creating a new repo:

![Screenshot 2019-08-27 at 13 54 52](https://user-images.githubusercontent.com/1822424/63773085-566a3880-c8d2-11e9-8230-a13ec31a772e.png)

The [docs on internal repos](https://help.github.com/en/articles/creating-an-internal-repository) seem to refer to enterprise instances, but `s/enterprise/organisation` still seems to match what that screenshot indicates about the behaviour on github.com.

Reply:
The docs on org settings seem to be lagging features on GitHub 😢

Reply:
Pushed a change.